packages/react-scripts/template/src/vendors.js

```diff
@@ -0,0 +1,11 @@
+// This file controls which npm packages are extracted into a vendor bundle.
+// A vendor bundle is very useful for reducing your transmitted bundle size,
+// as dependencies which rarely change can be cached while you iterate on your
+// application.
+
+// If you want to add another vendor, just add in this array the name of the
```

Comment:
This sounds backwards -- how about "If you want to add another npm package to the vendor bundle, just add its name in this array".

Reply:
I'm not fluent in english so... 😁

Reply:
No problem!